ITR/temperature_score.py

```diff
@@ -43,6 +43,26 @@ def __init__(self, time_frames: List[ETimeFrames], scopes: List[EScope],
         if grouping is not None:
             self.grouping = grouping
 
+    def _get_weighted_budget(self, q_s1s2: Quantity, q_s3: Quantity,
+                             scope: EScope, global_budget: Quantity) -> Quantity:
+        if q_s1s2 == None or np.isnan(q_s1s2.m) or \
+           q_s3 == None or np.isnan(q_s3.m):
+            return global_budget
+
+        if q_s1s2.units != q_s3.units:
```

Comment:
The whole point of Pint is that we can do unit conversions between compatible types.  We should lean on that, checking that types are *compatible*, not that they are equal.  We have data in kg, t, Mt, etc. , and should be able to keep it that way.

Reply:
this comment thread is outdated: in the latest push, i removed the commit, which raised the question in the first place.
Closing this thread as resolved